seqr/views/utils/orm_to_json_utils.py

```diff
@@ -555,3 +571,17 @@ def get_json_for_gene(gene, **kwargs):
     """
 
     return _get_json_for_model(gene, get_json_for_models=get_json_for_genes, **kwargs)
+
+
+def get_json_for_saved_searches(search, user):
+    def _process_result(result, search):
+        # Do not apply HGMD filters in shared searches for non-staff users
+        if not search.created_by and not user.is_staff and result['search'].get('pathogenicity', {}).get('hgmd'):
```

Comment:
to avoid code that expects a very specific search json structure, maybe better to just have separate sets of saved searches for staff / non-staff?